Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:alpine
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+RUN adduser -S sopel
+
+COPY requirements.txt /usr/src/app/
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
This `RUN pip ...` is no longer sufficient for sopel to run. `pyenchant` requires the `enchant` library, so a `apk add --no-cache enchant` will need to be added somewhere